### PR TITLE
modal custom content slots

### LINKED ISSUE #119: implement slots for modal customizability

### PR DESCRIPTION
**Pull request by chase-moskal**
closes #119

### CODE REVIEW

File: s/frontend/components/modals/install-modal-system.ts [hunk 1]
```diff
@@ -17,8 +17,14 @@ export function installModalSystem() {
...
-	const modals = <NamakaModals>document.createElement("namaka-modals")
-	document.body.prepend(modals)
+	const namakaModals = NamakaModals.elementName
+	let modals = <NamakaModals>document.querySelector(namakaModals)
+
+	if (!modals) {
+		console.log("create new modals")
+		modals = <NamakaModals>document.createElement(namakaModals)
+		document.body.prepend(modals)
+	}
```
Comment (zenkyuv) [inline s/frontend/components/modals/install-modal-system.ts:27]: i see now why my slots were not working

**Review by zenkyuv — APPROVED** [review]

File: s/frontend/components/modals/views/delete-thread/delete-post-modal-view.ts [hunk 2]
```diff
@@ -26,6 +26,7 @@ export async function deletePostModalView({
 		renderNo: () => html`Cancel`,
 		renderContent: () => html`
 		<div class="modalview deletepost">
+		<slot name=delete-post-lol-custom>Default Custom Text</slot>
```
[the remark this answers is not in the document]
Comment (chase-moskal) [reply s/frontend/components/modals/views/delete-thread/delete-post-modal-view.ts:29]: correct

i placed this custom text here just as an example,

please remove or change it when you are next working on this code :+1:

File: s/frontend/components/modals/install-modal-system.ts (hunk 1 above)
Comment (chase-moskal) [reply s/frontend/components/modals/install-modal-system.ts:27]: :+1:

**Comment by chase-moskal** [comment]
done in 7e7e2e0a8dc781f09c106675efb173880a68c491

**Pull request closed without merging**